tests/test_views.py

```diff
@@ -4582,6 +4528,246 @@ def test_confirm_user_signal_called_when_user_confirms_email(self):
 
         assert_equal(mock_signals.signals_sent(), set([auth.signals.user_confirmed]))
 
+class TestDraftRegistrationViews(OsfTestCase):
```

Comment:
TODO(@HarryRybacki): check that these tests conform to our new approvals patterns.